stratum/hal/bin/barefoot/deb/postinst

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
 # Copyright 2020-present Open Networking Foundation
 # SPDX-License-Identifier: Apache-2.0
-# Set up the huge page
-echo "vm.nr_hugepages = 128" >> /etc/sysctl.conf
-sysctl -p /etc/sysctl.conf
-mkdir -p /mnt/huge
-mount -t hugetlbfs nodev /mnt/huge
+
+# If not running in Docker,
+if ! cat /proc/1/cgroup | grep -q "/docker"; then
+  # Set up huge pages
+  echo "vm.nr_hugepages = 128" >> /etc/sysctl.conf
+  sysctl -p /etc/sysctl.conf
+  mkdir -p /mnt/huge
+  mount -t hugetlbfs nodev /mnt/huge
+fi
```

Comment:
The logic here would be that the page setup would always happen outside of docker?

Reply:
The problem is that we can't modify the host at deb install time, right?

Reply:
`sysctl` can't be used in Docker unless running with `--sysctl` or `--privileged` because the container and the host share the same kernel. Even if it could be run, it doesn't make sense to enable huge pages on the host building the image. Similar issue with `mount`...

Anyway, we need to enable huge pages on the switch manually each time the OS is installed, or we could add a check for huge pages in the `start-stratum.sh` script and do it there if needed.